ReplacePlaceholders.md

```diff
@@ -0,0 +1,69 @@
+# Replace Placeholders
```

Comment:
I'm still wondering about use cases. Is it basically a find-and-replace? It seems to be, because you can only really apply it once.